### ISSUE TITLE
Video is not working in Android Release build but wokrking fine in iOS

### ISSUE THREAD

**Issue by Shyama1201** [TheWidlarzGroup/react-native-video#2212]
# Bug

## Platform :- Android

Which player are you experiencing the problem on:
* Android

Video is not working in release android build but wokring fine with debug mode.

i tried with lower version of this sdk but getting blank black screen on release build in Android platform.

**Comment by saifbinabdulkarim** [comment]
I am also getting the same issue...

**Comment by cauyyl** [comment]
got the same issue +1

**Comment by official-bharat** [comment]
But I got the same issue on iPhone, not in android Device

**Comment by Curbe8** [comment]
Hello,

I was able to fix this issue by installing ``react-native-video@4.4.4`` and necessarily using ``react: "16.9.0"`` and ``react-native: "0.61.5"`` in the case of Android, for iOS, I can use the latest versions of ``"react": "16.13.1"`` and ``"react-native": "0.63.4"`` without problems.

Currently I could not find another way to use this plugin in the newer versions of react-native, so I carried out the project with 2 different versions, one for Android and one for iOS.

In the case of Android I had to follow the steps detailed in the readme, Installation section [here](https://github.com/react-native-video/react-native-video#android-installation)

I hope it helps more programmers looking for a solution to this plugin!
